### ISSUE TITLE
( ! ) [BUG] Furnitures not dropping in 1.156.3

### ISSUE THREAD

**Issue by MarkinhosMLP** [oraxen/oraxen#855]
### I have checked...

- [X] I am using the latest version of Oraxen
- [X] I am using the latest version of any dependencies
- [X] I am using ProtocolLib 5.0
- [X] I have checked if any similar bug reports exist

### Description

https://github.com/oraxen/oraxen/assets/47507632/1ddb6d95-0f56-4c04-82bc-ffc9aaece297
![image](https://github.com/oraxen/oraxen/assets/47507632/a82b747a-3514-4739-ae68-9bfe89552877)

### Steps to reproduc

place a furniture
break it

### Plugin Version

1.156.3

### ProtocolLib Version

5.0.0 b606

### Server Software & Version

MC 1.19.4 Purpur build 1968

### How breaking is the bug?

Non-breaking Bug - Plugin still usable, but certain features unavailable

**Comment by Boy0000** [comment]
this should be fixed in 1.156.4

**Comment by Boy0000** [comment]
ur config is malformed and does not have any items configured
```yml
drop:
  silktouch: false
  loots:
    - oraxen_item: chair
      probability: 1.0
```
drops is a list not one instance
